crates/astria-sequencer/Cargo.toml

```diff
@@ -17,13 +17,13 @@ benchmark = ["divan"]
 [dependencies]
 astria-core = { path = "../astria-core", features = ["server", "serde"] }
 astria-build-info = { path = "../astria-build-info", features = ["runtime"] }
+astria-eyre = { path = "../astria-eyre", features = ["anyhow_support"] }
```

Comment:
Please make a note here why this feature is necessary (because it's needed to implement the foreign trait `penumbra_ibc::component::HostInterface` in `crates/astria-sequencer/src/ibc/host_interface.rs`)